src/Microsoft.AspNetCore.WebHooks.Common/Extensions/WebHookConfigurationRootExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel;
+
+namespace Microsoft.Extensions.Configuration
+{
+    /// <summary>
+    /// Extension methods for <see cref="IConfigurationRoot"/>.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class WebHookConfigurationRootExtensions
```

Comment:
>WebHookConfigurationRootExtensions [](start = 24, length = 34)

We should not be adding things like, and certainly not in the webhooks repo

Reply:
The plan (#174) is to merge Common into the Receivers package. I could move this up into `WebHookReceiverConfigurationExtensions` to be proactive if you feel strongly about it.